### DISCUSSION TITLE
What is the minimum and recommended RAM requirements for openmptcprouter?

### DISCUSSION THREAD

**Discussion by ioogithub** [Ysurac/openmptcprouter#2506] [Q&A]
It looks like OpenWRT recommends "Sufficient RAM for stable operation: 32MB min, 64MB better" according to https://openwrt.org/supported_devices

What is the minimum and recommended RAM requirements for [openmptcprouter](https://github.com/Ysurac/openmptcprouter). Does it benefit from using a device with more RAM or would a device with higher ram be wasted in this application?

I am considering using a raspberry pi 4b so the choices are 1GB, 2GB, 4GB and 8GB.

**Comment by Ysurac** [comment]
It doesn't need too much RAM. When v2ray is used as proxy it use a bit more RAM.
I have a RPI4 with 2GB and less than 512MB is used.

RPI4 are expensive for now, some x86_64 barebone may be cheaper and better (but it's less easy to do initial install)
